java/.gitignore

```diff
@@ -12,3 +12,4 @@ dependency-reduced-pom.xml
 .tested*
 .compared
 .built
+src/main/resources/**/ciDict.json
```

Comment:
You don't need this one if you copy the json to `${project.build.directory}/io/cucumber/createmeta`